src/Rule.php

```diff
@@ -1538,6 +1538,15 @@ public function process(&$input, &$output, &$params, &$options = [])
                 $hook_params["output"]   = $output;
                 Plugin::doHook(Hooks::RULE_MATCHED, $hook_params);
                 $output["_rule_process"] = true;
+
+                //If the rule has changed the ticket status, add a param to avoid another status change
+                if ($this instanceof RuleTicket) {
+                    foreach ($this->actions as $action) {
+                        if ($action->fields['field'] === 'status') {
+                            $output['_status_changed'] = true;
```

Comment:
`isStatusComputationBlocked()` will check `_do_not_compute_status`

```suggestion
                            $output['_do_not_compute_status'] = true;
```

Reply:
Thank you! applied :+1: [9573779](https://github.com/glpi-project/glpi/pull/17088/commits/95737792deeb55180829debceba54ebb198d8223)